src/simulcastlayers/SimulcastLayers.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * [[SimulcastLayers]] represents simulcast layers for selected simulcast video streams.
+ */
+export enum SimulcastLayers {
+  /**
+   * Low video stream resolution: 320x180@15fps.
```

Comment:
We can't guarantee that these will be the exact resolutions of the low, medium and high.  Can you  remove the resolution and frame rate information?

Reply:
Ok, thanks for catching this. I have updated it now.